### ISSUE TITLE
seawreck does not with DPDK UP

### ISSUE THREAD

**Issue by slivne** [cloudius-systems/seastar#31]
```
sudo build/release/apps/seawreck/seawreck --server 192.168.20.101:10000 --host-ipv4-addr 192.168.20.185 --dhcp 0 --gw-ipv4-addr 192.168.20.101  --netmask-ipv4-addr 255.255.255.0 --smp 1 --conn 4 --network-stack native --dpdk-pmd  --collectd 0  --duration 1
```
with 8ca0f21ae63e813383c64d6459cb29f554dd2a5d

this works on the same server with posix stack (--network-stack posix)

throws an assert

this is the output

ports number: 1
Port 0: max_rx_queues 316 max_tx_queues 316
Port 0: using 1 queue
RX checksum offload supported
TX ip checksum offload supported
TX TCP&UDP checksum offload supported
Port 0 init ... done: 
Creating Tx mbuf pool 'dpdk_net_pktmbuf_pool0_tx' [1024 mbufs] ...
Creating Rx mbuf pool 'dpdk_net_pktmbuf_pool0_rx' [1536 mbufs] ...
PMD: i40e_dev_rx_queue_setup(): Rx Burst Bulk Alloc Preconditions are satisfied. Rx Burst Bulk Alloc function will be used on port=0, queue=0.
PMD: i40e_dev_tx_queue_setup(): Using full-featured tx path

Checking link status 
Created DPDK device
.done
Port 0 Link Up - speed 40000 Mbps - full-duplex
========== http_client ============
Server: 192.168.20.101:10000
Connections: 4
Requests/connection: dynamic (timer based)
seawreck: net/dpdk.cc:219: virtual unsigned int dpdk::dpdk_device::hash2qid(uint32_t): Assertion `_redir_table.size()' failed.

this is similar to the issue I reported for running tcp_client with dpdk (https://github.com/cloudius-systems/seastar/issues/22)

**Comment by slivne** [comment]
@gleb-cloudius I tested head with your commit 1cfaa7eefef4825216d6ad24e1d1efc40de1d83a it fixes this issue - can we close this ?

**Comment by gleb-cloudius** [comment]
On Tue, Feb 24, 2015 at 12:46:02AM -0800, slivne wrote:
> @gleb-cloudius I tested head with your commit 1cfaa7eefef4825216d6ad24e1d1efc40de1d83a it fixes this issue - can we close this ?
> 
Yes, we can.

--
			Gleb.

**Comment by slivne** [comment]
closed by 1cfaa7eefef4825216d6ad24e1d1efc40de1d83a
